src/components/EmailVerification/SendCodeButton.js

```diff
@@ -0,0 +1,64 @@
+import {useFormikContext} from 'formik';
+import PropTypes from 'prop-types';
+import {useContext, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {ConfigContext} from 'Context';
+import {post} from 'api';
+import {OFButton} from 'components/Button';
+import Loader from 'components/Loader';
+
+const createVerification = async (baseUrl, {submissionUrl, componentKey, emailAddress}) => {
+  await post(`${baseUrl}submissions/email-verifications`, {
+    submission: submissionUrl,
+    componentKey,
+    email: emailAddress,
+  });
+};
+
+const SendCodeButton = ({submissionUrl, componentKey, emailAddress, onError}) => {
+  const {baseUrl} = useContext(ConfigContext);
+  const [isSending, setIsSending] = useState(false);
+  const {setFieldValue} = useFormikContext();
+  return (
+    <OFButton
```

Comment:
yeah disabling that verify button is not possible because formio makes shit way harder than it should be, so the next best option is disabling this button, but then it has to work of course :sweat_smile: I'll investigate and fix it!

Reply:
Fixed!